### DISCUSSION TITLE
How to load a text file?

### DISCUSSION THREAD

**Discussion by changfeng2021** [nuxt/nuxt#20385] [Questions] [answered]
hi 
Please tell me how to load text file in nuxt3 js file?
like `~/utils/string.ts load ~/file/simple.txt`

**Comment by localusercamp** [comment]
Describe in more detail what behavior you want.

**Reply by changfeng2021** [reply]
> Describe in more detail what behavior you want.

I want to import text file inside js file.
For example `import rawTxt from '@/md/article.md'`
please tell me how can i do this

**Reply by localusercamp** [reply]
Do you want to import it like string value for a variable? If so, then you need to use 'fs' like so:
```typescript
import { readSync } from 'fs';

const content = readSync('@/md/article.md').toString();
```
But it works only for server-side (node)

**Reply by changfeng2021** [reply]
Thanks for the reply, but I want to import txt on client-side ,I will use nuxt to generate static pages

**Comment by changfeng2021** [comment, marked as answer]
just use `import file from '~/assets/md/java.md?raw'`
